tests/unit/resources/test_attachment.py

```diff
@@ -24,26 +24,26 @@ def attachments_tester(attachment):
 
 
 @responses.activate
-def test_should_get_attachment(ENDPOINT_BASE, attachment):
+def test_should_get_attachment(client, ENDPOINT_BASE, attachment):
     responses.add(
         responses.GET,
         ENDPOINT_BASE + f"/experiments/{EXP_ID}/attachments",
         json=[attachment.to_dict()],
     )
-    att = Attachment.get(EXP_ID, attachment._id)
+    att = client.get_attachment(EXP_ID, attachment._id)
```

Comment:
Relates to https://github.com/primitybio/cellengine-python-toolkit/pull/112#discussion_r758884306

It looks like we still have `Attachment.get/upload` now, but they're no longer tested. Copy these tests to test both?

Reply:
:+1:

Reply:
Looks like you still need one for `Attachment.upload`; only `client.upload_attachment` is tested.